### DISCUSSION TITLE
Update to DS18x20 documentation

### DISCUSSION THREAD

**Discussion by Silverfire9** [arendst/Tasmota#18668] [General]
I was just looking in the DS18x20 documentation to see if there's any sort of temperature reading validation published and didn't find it. That's fine - no problem there, but while looking I noticed that in the wiring section, the wired version has only a black wire listed for ground. I have never seen one with a black ground wire. While haven't gotten many - probably under a dozen - I've gotten sensors from Adafruit, Element14, and Wish over the past 3-4 years so it's not an isolated source with the new wiring. Mine have all had red, yellow and blue wires. I'm sure there are still sensors out there with black wires, but it's probably worth the effort to add the blue option since they seem to be becoming relatively widespread.

In case it helps - a link to the page in question:[https://tasmota.github.io/docs/DS18x20](url)

**Comment by Jason2866** [comment]
Tasmota Docu is OpenSource too. Just click the pencil in upper right corner

**Comment by lalo-uy** [comment]
All sensors I got from Ali ( cuple of 100’s in dozen of different buys) are
either red/yelow/black or red/blue/black.
So there seems no color standard.
